designs/administrator/olm/clarify-operatorhub-source-support/index.md

```diff
@@ -0,0 +1,34 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Clarify OperatorHub sources and support
+
+Today OperatorHub includes 4 default catalog sources, which provide different sources of operators. It’s not uncommon for a partner’s operator to have a Community version, a Certified version, and a Marketplace version, each showing up individually in OperatorHub. Community operators include a badge to indicate they are community-contributed, and not supported by Red Hat. Similarly, Marketplace have badges to indicate they are the Marketplace version of that operator that requires purchase from the RHM and includes additional support.
+
+Because Red Hat and Certified operators do not have badges or indications on the tile of what their source is, it can be left to the user to try to infer what these operators are. Additionally, users may not completely understand the difference between Red Hat/Certified operators and their Marketplace equivalents, leaving them unsure which version to install.
+
+With this design, OperatorHub will include additional information to convey to users what the source of operators are, and what that source represents to them (generally a different level of support and cost, in the case of Marketplace.)
+
+## Source tile badge and field level help
+
+By adding the Catalog Source name to each tile, this will help the user better understand that this piece of information corresponds to the Catalog Source that operator comes from (vs just indicating it as a Community or Marketplace operator.) The Catalog Sources can then be further described using field level help near the Sources filter, to better convey the difference between the sources.
+
+Additionally, this will also help differentiate between when the same operator appears many times from different sources (such as Catalog Sources for nightly builds of an operator) with that information appearing right on the tile and further described in a tooltip.
+
+
+![OperatorHub tiles](img/1-1a.png)
```

Comment:
thinking more about this and how it aligns with Virtualization templates. Have you thought about the confusion between Provided vs Source? We have had the same issue with Provided vs Supported (in our case supported is the same as a source here). We had users telling us it was very hard to know the difference.

Reply:
This actually was maybe even more confusing until recently, though that's not to say it's now absolutely clear. Previously the filters were "Provider Type" for selecting the CatalogSource and "Provider" for choosing the provider. We had [some feedback](https://coreos.slack.com/archives/C6A3NV5J9/p1610571370405000?thread_ts=1610570799.403800&cid=C6A3NV5J9) that it wasn't always clear that the "Provider Type" represented the CatalogSource so [recently changed that heading](https://bugzilla.redhat.com/show_bug.cgi?id=1919406) to "Source," to closer tie it to the CatalogSource term without being too specific since which precise "source" resource it represents may change in the future.

I am hoping now with the "Source" popover having field level help that conveys it contains CatalogSources, and the facets under Provider appearing to be various providers/authors, that maybe the difference between the two is clearer (though still requires some amount of inference.)

Reply:
ya, that's certainly an improvement.